content/post/083-js-ipfs-0.41.md

```diff
@@ -0,0 +1,209 @@
+---
+date: 2020-02-14
+url: 2020-02-14-js-ipfs-0-41
+title: js-ipfs 0.41.0 released
+author: Alan Shaw
+---
+
+# 🔦 Highlights
+
+> One giant leap forward
+
+## 🦁 Async Await and Async Iterables
+
+🎶 In the jungle, the mighty jungle the lion sleeps tonight!
+🎶 async await, async await, async await, async await...
+
+We've completed a [**HUGE** refactor](https://github.com/ipfs/js-ipfs/issues/1670) to js-ipfs internals 🥳, switching to using Promises and `async`/`await` over Callbacks and using async [iterables](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols) instead of [Node.js](https://nodejs.org/dist/latest/docs/api/stream.html) and [Pull Streams](https://pull-stream.github.io/). Ok, tell a lie, it's actually not just js-ipfs internals, it's the whole stack, including libp2p, IPLD and multiformats, you might call it a ground up re-write, but, you know, we don't like to brag.
+
+🚨 Oh, wait, serious note - this release brings **big breaking changes** to the core API so please consult the "API Changes" section below for all the information.
```

Comment:
```suggestion
🚨 Oh, wait, serious note - this release brings **big breaking changes** to the core API so please consult the "[API Changes](#-api-changes)" section below for all the information.
```

Reply:
I think...in hugo, this will be `#api-changes`

Reply:
Ah, right! Fixed.